cloudify_agent/installer/__init__.py

```diff
@@ -259,7 +259,7 @@ def install_pip(self):
         get_pip_url = 'https://bootstrap.pypa.io/get-pip.py'
         self.logger.info('Downloading get-pip from {0}'.format(get_pip_url))
         destination = '{0}\\get-pip.py'.format(self.cloudify_agent['basedir'])
-        get_pip = self.download(get_pip_url, destination)
+        get_pip = self.runner.download(get_pip_url, destination)
```

Comment:
we need to skip our internal cert mechanism because this just uses an external pypa.io url